torch/csrc/jit/passes/quantization.cpp

```diff
@@ -1875,7 +1860,14 @@ ModuleMethodVector InsertQuantDeQuantHelper::getInvokedMethods(
         if (module_instance == graph->inputs()[0]) {
           m = module;
         } else {
-          m = findChildModuleToQuantize(module, module_instance);
+          TORCH_INTERNAL_ASSERT(
+              module_instance->node()->kind() == prim::GetAttr,
+              "Module instance should come from GetAttr.");
```

Comment:
this is checked in `getInvokedModule` already, you can remove it.

Reply:
oh, actually this is needed since we are checking the child instance name, but maybe make this a condition instead.

Reply:
Why condition? Don't we want to assert here?

Reply:
this insert will actually reject all models that pass instances in arguments:
e.g.
```python
def forward(self, m):
       m.func(...)
```

Reply:
So we were asserting incorrectly before? Since we passed in `module_instance` to the older function,

Reply:
yeah I think so, but we don't have such test cases though, this is not a common use case.